### DISCUSSION TITLE
OV 5640

### DISCUSSION THREAD

**Discussion by s0170071** [jomjol/AI-on-the-edge-device#1529] [Q&A]
Hi all, 
I just discoverd a new ESP32 CAM module. It comes with a USB-c connector and the 5MP camera OV 5640. 

https://de.aliexpress.com/item/1005004521039608.html
 
![image](https://user-images.githubusercontent.com/33855380/206664500-88404975-6265-4757-9110-46ff4d75cb55.png)

Has anyone of you tried it ?

**Comment by jomjol** [comment]
It looks like it has no PSRAM --> will not work.

**Reply by s0170071** [reply]
Seems to be identical to this one and here it says psram in the description
https://m.de.aliexpress.com/item/1005004567918690.html?gatewayAdapt=gloPc2deuMsite

I found plenty of other sellers that also advertise psram. Sometimes 8Mb. That is more than the esp can address.
What I didn't see was a LED.
